singleton-service/src/service/proxy.rs

```diff
@@ -153,10 +153,11 @@ impl RootContext for SingletonService {
         .cloned()
         .collect();
         let threescale1 = ThreescaleData {
-            app_id: "46de54605a1321aa3838480c5fa91bcc".to_string(),
-            service_id: "2555417902188".to_string(),
-            service_token: "6705c7d02e9a899d4db405dc1413361611e4250dfd12ec3dcbcea8c3de7cdd29"
-                .to_string(),
+            app_id: AppIdentifier::UserKey(UserKey::from("46de54605a1321aa3838480c5fa91bcc")),
+            service_id: ServiceId::from("2555417902188"),
+            service_token: ServiceToken::from(
+                "6705c7d02e9a899d4db405dc1413361611e4250dfd12ec3dcbcea8c3de7cdd29",
```

Comment:
These are still hanging around here, let's try and find a way around hardcoding them.

Reply:
@unleashed will get fixed from my next PR.